QA-CHECKLIST.md

```diff
@@ -11,7 +11,7 @@
 When going through the following checklist, keep the browser console open. There should not be any errors in the console!
 
 a rigourous check will verify that the items listed below work as expected in:
-1. localhost in client only mode (`npm start-and-serve`)
+1. localhost in client only mode (`npm simple-serve`)
```

Comment:
This is `npm run simple-serve` everywhere else... should it be here?

Reply:
Oops, yes! Good check, will fix before landing